CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
 cmake_minimum_required(VERSION 3.5)
 project(cppJNI)
+set(CMAKE_CXX_STANDARD 14)
 
+# Extend CMAKE_PREFIX_PATH with HanaConfig.cmake and Gtest root
+#list(APPEND CMAKE_PREFIX_PATH "")
+#SET(GTEST_ROOT "")
```

Comment:
I don't like commented out code, please remove it: such configuration shouldn't be a part of project cmake files, so uncommenting it doesn't make sense IMO.